experimental/language-generation/csharp_dotnetcore/05.multi-turn-prompt/AdapterWithErrorHandler.cs

```diff
@@ -2,12 +2,13 @@
 // Licensed under the MIT License.
 
 using System;
+using System.IO;
 using Microsoft.Bot.Builder;
-using Microsoft.Bot.Builder.Integration.AspNet.Core;
-using Microsoft.Bot.Connector.Authentication;
 using Microsoft.Extensions.Logging;
+using Microsoft.Bot.Connector.Authentication;
 using Microsoft.Bot.Builder.LanguageGeneration;
-using System.IO;
+using Microsoft.Bot.Builder.Integration.AspNet.Core;
+using ActivityBuilder = Microsoft.Bot.Builder.Dialogs.Adaptive.Generators.ActivityGenerator;
```

Comment:
If we don't mind take daily package, this can be "ActivityFactor" now. we don't have to use this "using"